tests/unit/ext/switches/brocade.py

```diff
@@ -20,77 +20,77 @@
 from haas.ext.obm.ipmi import Ipmi
 
 MODE_RESPONSE_ACCESS = """
```

Comment:
That's annoying.

The other problem here is that doing this modifies the string; `#` doesn't get stripped out if its inside a literal. I haven't looked at this closely enough to  know if that matters for these tests in particular.

Options:

* Leave it as is
* move the string assignments to their own file, with no other code, and ignore the whole file. Add a comment to the top to the effect that this file is not checked and, and so be careful, don't put much code here, yada yada.
* Put the xml in separate files (ending in `.xml`) rather than being python source.

I can see arguments for each. If we have a concrete reason why adding `# noqa` to the content of that xml isn't a problem, then probably the best thing is to add a comment near the top of the file explaining the whole business, and just leave it as is.